### DISCUSSION TITLE
Possibility for a Snapshot Version?

### DISCUSSION THREAD

**Discussion by Cazitron** [maruohon/litematica#315] [Q&A]
I mean, my question is in the title but is it difficult to update the Litematica mod to run on a snapshot. I'm pretty new to how Fabric Mods work, but I don't think there have been any changes to modding capabilities, so wouldn't an update to a snapshot be possible? Please do fill me in if this is the case or not, and I wouldn't mind knowing why as well.

**Comment by maruohon** [comment]
There are already ports of my client mods to the snapshots. I don't update to every snapshot, as that's just a waste of my time for something that sometimes becomes obsolete the very next week when something changes and the previous versions won't work anymore...

Currently Litematica has been updated up to 21w08b, but 21w10a changed some fundamental rendering stuff again and it won't work on that version anymore, and currently I don't plan to update to that snapshot, for the above mentioned reason.

They are available here: https://masa.dy.fi/mcmods/client_mods/ (this is also linked on the CurseForge page),
